Use short labels for the legend tests so as not to be affected by the font width

nagix opened the pull request:
This fixes the #5816 test failures that can happen depending on environments.

nagix commented:
All the text labels for the label box position tests are changed to empty. @simonbrunel Can you test again?

nagix commented:
Well, they test the size and positions of `legendHitBoxes` which consist of a sample box + a padding area + a text label box. If we check if the size of the text label box is reflected in `legendHitBoxes`, would it be better to use some simple string like `' '`?

simonbrunel commented:
I think it will be the same issue with `' '` but we can try (though, that's weird it doesn't generate the same width for a space character). We can also try to use a more basic / common / generic font family in our tests (e.g. `'Arial'`). Default is `'Helvetica Neue', 'Helvetica', 'Arial', sans-serif` but I'm on Windows and don't have Helvetica Neue or Helvetica installed.

nagix commented:
I tried Arial with different browsers on both Mac and Windows, but the results are not consistent at all ([reference](https://stackoverflow.com/questions/28740918/different-font-widths-in-html5-canvas)). Eventually, I reverted the changes in labels and added some adjustment to pixel values so that differences across environments stay within the margin range (+/-2).

nagix commented:
It was passed on Mac and Windows, but still fails on Linux.

nagix commented:
Now all the test passed. The results were more consistent when `' '` is used for labels.
